dll/win32/msgina/shutdown.c

```diff
@@ -538,6 +561,116 @@ ShutdownDialog(
     return ret;
 }
 
+/*************************************************************************
+ * SetRegistryToBootInNTNativeMode
+ *
+ * NOTES
+ *     Used to append the NT Native shell start-up arguments to the
+ *     BootExecute registry value so that the system will boot into it.
+ */
+VOID SetRegistryToBootInNTNativeMode(VOID)
+{
+    #if 1
```

Comment:
The dead code is going to confuse compilers & static analyzers. You could e.g. `#if 0` out the contents of the function instead.

Reply:
Inverted #if logic. Ideally I could add a check in GetAllowedShutdownOptions() to check if the user is an administrator and/or the CTRL key is pressed but that probably warrants its own separate discussion I think.